### PR TITLE
[Testing screener] Reduce dom for Persona

### PR DESCRIPTION
**Pull request by Markionium**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by dzearing** [comment]
change file

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx [hunk 1]
```diff
@@ -108,23 +96,7 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
       <div {...divProps} className={classNames.coin}>
```
Comment (lju) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: It looks like this component is still a few levels deep. For the scenario in which we need just the image or the initials, can we make it one div? Or create a separate component that's just one div?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Yes, like https://codepen.io/dzearing/pen/BVjzRW

Comment (Markionium) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Yep, i basically put the PR here to have screener run on my commits, so i can follow i'm not really breaking anything.

I'll pick this up today and see if i can reduce the DOM impact it further :)

Comment (lju) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Okay please let me know how I can help!

**Comment by dzearing** [comment]
Given this is becoming stale, I will close; presumably, what we should do is build an experimental replacement. If you move it to experiments, you can make breaking changes and then we can just move the folder back over in the next major update.

**Pull request closed without merging**